scan.go

```diff
@@ -220,6 +228,12 @@ func (zp *ZoneParser) SetIncludeAllowed(v bool) {
 	zp.includeAllowed = v
 }
 
+// SetIncludeAllowedFS controls whether $INCLUDE directives are allowed, and
+// specifies an [fs.FS] from which to open and read files.
+func (zp *ZoneParser) SetIncludeAllowedFS(v bool, fsys fs.FS) {
```

Comment:
On the fence if we need a whole new method for this. Should we give SetIncludeAllowed an optional fs.FS parameter instead?

Reply:
Shipping separate methods as a non-breaking change, and then consolidating them if/when you release a v2 is an option.

Reply:
ack. Think my main objection is the overlap in functionality. Maybe a better idea is to have a seperate method to set the underlying FS? `SetIncludeFS` that just sets the FS